maven/bnd-maven-plugin/README.md

```diff
@@ -61,7 +52,7 @@ pom. Bnd instructions in the pom are not used if the project has a bnd file.
  org.example.api,\
  org.example.types
 -sources: true
-]]></bnd>
+        ]]></bnd>
```

Comment:
It is wrong to show the `<bnd>` example lines having leading spaces. Bnd will interpret this as a line continuation.

Reply:
works perfectly in practice...